InterviewFlashCards/QueryManager.swift

```diff
@@ -1,12 +1,11 @@
-
 import Foundation
 import Firebase
 import FirebaseDatabase
 
-public typealias JSON = [[String: Any]]
-public protocol Networking {
+public typealias JSON = [String: Any]
```

Comment:
So now `JSON` represents `[String:Any]` and when you put  `[]` around JSON =>`[JSON]`, it then represents `[[String:Any]]`